requirements-dev.txt

```diff
@@ -45,7 +45,7 @@ html5lib
 lxml
 openpyxl
 pyarrow>=0.9.0
-pyqt
+pyqt5>=5.9.2
```

Comment:
how recent is this?

Reply:
recent `pyqt5` in pypi is 5.13.0, while `pyqt` in conda is 5.9.2, since in order to run this `generate_pip_deps_from_conda.py`, version has to be added for `pyqt`, so 5.9.2 is generated here for `pyqt5` @jreback